mkdocs.yml

```diff
@@ -31,7 +31,7 @@ nav:
         - From local databases: 'guides/adding-data/local-DB.md'
         - From JSON files: 'guides/adding-data/JSON.md'
         - From the web: 'guides/adding-data/external-uri.md'
-      - Viewing data (Data Context): ''
+      - Data Context: 'guides/viewing-data.md'
```

Comment:
I'm not sure what we should name this section, because data context is its name yet it doesn't sound very intuitive.